src/assets/css/main.css

```diff
@@ -43,109 +43,147 @@ body{
     user-select: none;
 }
 
-#buttons {
-    display: grid;
-    grid-template-columns: 1fr 1fr 1fr 1fr;
-}
+#calculator-buttons{
+    width: auto;
+    height: auto;
+    border-radius: 0 0 8px 8px;
 
-#buttons button {
-    padding: 20px 0px 20px 0px;
+}
+button{
```

Comment:
add this css only for buttons inside of calculator

for example

```css
#caculator button
```

Reply:
it's done